backend/processing/check_files_checksums/task.py

```diff
@@ -0,0 +1,10 @@
+"""
+TODO: add docstring here.
```

Comment:
We don't need these anymore.

Reply:
Fixed in a954b72ed0860ce9120aff8d3f1457456f2932c3